.github/workflows/tests.yml

```diff
@@ -0,0 +1,32 @@
+name: "Tests"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
```

Comment:
I'm not sure but I think we can move to something like that.;
```
 on:
  pull_request:
    branches:
      - "master"
```

Reply:
As it is, it also runs the tests when a pull request is merged into the master